From 500cd53034ab520a03ed79061639bfc89aa47956 Mon Sep 17 00:00:00 2001
From: SeongJae Park <sj@kernel.org>
Date: Sat, 22 Jun 2024 15:56:43 -0700
Subject: [PATCH] Docs/process/index: Remove unaligned-memory-access from
 'Other material'

unaligned-memory-access document is well organized into core-api/ and
linked on the core-api/index.  Remove it from 'Other material' section
of process/index.

Signed-off-by: SeongJae Park <sj@kernel.org>
---
 Documentation/process/index.rst | 1 -
 1 file changed, 1 deletion(-)

diff --git a/Documentation/process/index.rst b/Documentation/process/index.rst
index de9cbb7bd7eb..f66f9cbd0300 100644
--- a/Documentation/process/index.rst
+++ b/Documentation/process/index.rst
@@ -116,7 +116,6 @@ lack of a better place.
    magic-number
    clang-format
    ../arch/riscv/patch-acceptance
-   ../core-api/unaligned-memory-access
 
 .. only::  subproject and html
 
-- 
2.39.2

